src/dr_llm/llm/catalog/fetchers/__init__.py

```
@@ -0,0 +1,62 @@
+from dr_llm.llm.providers.anthropic.provider import AnthropicProvider
+from dr_llm.llm.providers.google.provider import GoogleProvider
+from dr_llm.llm.providers.headless.claude import ClaudeHeadlessProvider
+from dr_llm.llm.providers.headless.codex import CodexHeadlessProvider
+from dr_llm.llm.providers.kimi_code import KimiCodeProvider
+from dr_llm.llm.providers.minimax import MiniMaxProvider
+from dr_llm.llm.providers.openai_compat.provider import OpenAICompatProvider
+from dr_llm.llm.providers.base import Provider
+from dr_llm.llm.catalog.models import ModelCatalogEntry
+
+
+def fetch_models_for_provider(
```

Ruff (0.15.9)

[warning] Too many return statements (7 > 6)

(PLR0911)